static/.htaccess

```diff
@@ -652,7 +652,7 @@ RewriteRule ^docs/reference/tasklist-api(.*)$ /docs/apis-clients/tasklist-api$1
 RewriteRule ^docs/reference/tasklist-api$ /docs/apis-clients/tasklist-api [R=301,L]
 
 RewriteRule ^docs/reference/public-api/$ /docs/apis-clients/public-api [R=301,L]
-RewriteRule ^docs/reference/grpc/$ /docs/apis-clients/grpc [R=301,L]
+RewriteRule ^docs/reference/grpc/?$ /docs/apis-clients/grpc/ [R=301,L]
```

Comment:
This looks like a very old redirect, before we added a trailing slash.

@pepopowitz would it make sense to audit the other rules without a trailing slash too?

Reply:
We definitely could, but I personally feel like that's a task that will sit on our board due to low priority and we'll never get to it. If you're feeling ambitious though, go for it!